plugins/modules/azure_rm_virtualnetwork_info.py

```diff
@@ -225,6 +225,8 @@ def __init__(self):
             changed=False,
             virtualnetworks=[]
         )
+        
+        self.required_together = [('name', 'resource_group')]
```

Comment:
If get all VNT by resource group, it will get fail.
```suggestion
        self.required_if = [('name', '*', ['resource_group'])]
```